packages/grafana-runtime/src/config.ts

```diff
@@ -238,16 +235,32 @@ function overrideFeatureTogglesFromLocalStorage(config: GrafanaBootConfig) {
   }
 }
 
-function overrideFeatureTogglesFromUrl(config: GrafanaBootConfig) {
+function overrideFeatureTogglesFromUrl(config: GrafanaBootConfig, allowWhiteListed = false) {
   if (window.location.href.indexOf('__feature') === -1) {
     return;
   }
 
+  const isLocalDevEnv = config.buildInfo.env === 'development';
+
+  const prodUrlAllowedFeatureFlags = new Set([
+    'autoMigrateOldPanels',
+    'autoMigrateGraphPanel',
+    'autoMigrateTablePanel',
+    'autoMigratePiechartPanel',
+    'autoMigrateWorldmapPanel',
+    'autoMigrateStatPanel, disableAngular',
```

Comment:
Maybe I am missing something but shouldn't this be:

```suggestion
    'autoMigrateStatPanel',
    'disableAngular',
```

Reply:
good catch, yes it should!